src/Integration.Vsix/Telemetry/GeneratedCode/SqmOleCommandTarget.cs

```diff
@@ -33,6 +33,7 @@ internal enum SonarLintSqmCommandIds
         ToggleShowAllProjectsCommandCommandId = 0x306,
         DontWarnAgainCommandCommandId = 0x307,
         FixConflictsCommandCommandId = 0x308,
+        UpdateBindingCommandFromErrorListCommandId = 0x400,
```

Comment:
That is the point that your commands are real commands and not SQM fake commands that are there in order to have telemetry. You will have telemetry for your commands any way. That separation and the fact that we don't really look in the generated code for duplicates with the real commands suggests that you need a separate command set to avoid conflicts etc. That is my opinion, without checking what is the common practice (which may not take conflicts into account).
As to why - just a common practice to have logical groups of commands that I've picked up, so you can think about a general pattern that is being used in VS which I used here. The context is a bit different there since you do need to look and search for command ids every now and then, unlike in the SQM case in which you don't.